source/shared/cpp/ObjectModel/HostConfig.h

```diff
@@ -407,13 +407,14 @@ struct TableConfig
 
 struct BadgeConfig
 {
-    std::string backgroundColor = "#DCECFC";
+    std::string backgroundColor = "#5B5FC7";
```

Comment:
Will this change be expected on Teams host config also or limited to SDK?

Reply:
This colour change is due to change in design.  For dark mode, we will need changes in hostConfig as well , for light mode default colour can be used